lute/templates/read/index.html

```diff
@@ -370,27 +414,27 @@ <h2>&#127881;</h2>
     const menu = document.getElementById("reading_menu");
     menu.classList.remove("open-menu");
 
-    disableScroll(false);
+    // disableScroll(false);
   }
 
   function toggleMenu() {
     const menu = document.getElementById("reading_menu");
-    menu.classList.toggle("open-menu");
+    menu.classList.add("open-menu");
 
-    disableScroll(true);
+    // disableScroll(true);
```

Comment:
No disabling scroll needed anymore?  Should we just remove this code and related code instead of commenting it out, or are you unsure about this?

Reply:
after making the `reading_menu ` fixed not disabling scroll gives better results. menu is fixed, but you can still scroll the text and not add a scrollbar which was annoying. I am just doubting myself sometimes and keeping code commented until next update to see if weird bugs come out